internal/app/orchestrator/orchestrator.go

```diff
@@ -76,15 +81,18 @@ type orchestrator struct {
 func New(
 	ctx context.Context,
 	l log.Logger,
+	scope tally.Scope,
 	mapper mapper.Mapper,
 	upstreamClient upstream.Client,
 	cacheConfig *bootstrapv1.Cache,
 ) Orchestrator {
 	orchestrator := &orchestrator{
 		logger:                l.Named(component),
+		scope:                 scope,
 		mapper:                mapper,
 		upstreamClient:        upstreamClient,
 		downstreamResponseMap: newDownstreamResponseMap(),
+		downstreamScope:       scope.SubScope("downstream"),
```

Comment:
Ah I meant can you pass this to the above line `newDownstreamResponseMap(scope.Subscope("downstream"))`?

And have the `createChannel` func do the stat increment?

Reply:
oh my goodness, obviously. What was I thinking? 😅